torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -442,6 +527,24 @@ class CppMicroGemmFP32Vec(CppMicroGemm):
         }
 
         if constexpr (row == 0) {
+{%- if tail_n %}
+            if (col < rCOLS) {
+{%- if input2_dtype in [torch.bfloat16, torch.float16] %}
```

Comment:
Can we properly indent these template control structures for better readability?

Reply:
Thanks for your comments. Revised.